packages/react-core/src/components/Banner/examples/BannerStatus.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Banner } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle-icon';
+import InfoCircleIcon from '@patternfly/react-icons/dist/esm/icons/info-circle-icon';
+import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+
+export const BannerStatus: React.FunctionComponent = () => (
+  <>
+    <Banner>
+      <BellIcon /> Default banner
```

Comment:
This looks fine, but in the core examples we added a `<Flex>` layout to handle the icon/text spacing. The whitespace character makes the space a little tight.

Is that OK to add here, or do you think it's confusing, since users might think the flex layout is needed when we're just providing an example to show the ideal spacing?